pkg/extensions/tbtc/tbtc_test.go

```diff
@@ -1538,6 +1561,7 @@ func TestProvideRedemptionProof_ContextCancelled_WithWorkingMonitoring(
 		ctx,
 		constantBackoff,
 		timeout,
+		defaultLocalBlockConfirmations,
 	)
 	if err != nil {
 		t.Fatal(err)
```

Comment:
It would be good to have tests validating monitoring goroutines are really confirming the state on-chain. We may need to do it in another file with some short block times to do not hang on those tests too long.

Reply:
Those tests are really confirming the state. They are just doing this after `0` blocks.

Reply:
Correct but if I completely remove the code calling the confirmation function, tests will still pass, correct?

Reply:
I understand your point. Those tests will still pass in that case but from the other side, confirmations make sense only when used with Ethereum. Writing separate tests for them using the local chain will say us the local chain is correctly implemented and nothing more.

Reply:
Unless we increase the block time for a local chain and... assert on time? OK, I do agree it's one of those rare cases where it's probably not worth it.